doc/lua_api.txt

```diff
@@ -6092,6 +6092,9 @@ object you are working with still exists.
           (default: `false`)
         * `new_move`: use new move/sneak code. When `false` the exact old code
           is used for the specific old sneak behaviour (default: `true`)
+        * `new_jump`: use the new jump/bouncy jump code. When `false` the exact
```

Comment:
jump/bouncy jump? I don't think this was intended?

Reply:
It was, you can now continuously jump on bouncy nodes